### PR TITLE
Fixing page layouts

### PR DESCRIPTION
**Pull request by nhejazi**
Discussion of figure placement in `bookdown`: https://bookdown.org/yihui/rmarkdown-cookbook/figure-placement.html

### CODE REVIEW

File: cor_coxph/code/params.R [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # B=1e3 and numPerm=1e4 take 10 min to run with 30 CPUS for D57 CoR
-B <- 1e3 # number of bootstrap replicates 1e3
-numPerm <- 1e4 # number permutation replicates 1e4
+B <- 5 # number of bootstrap replicates 1e3
```
Comment (benkeser) [inline cor_coxph/code/params.R:2]: Revert?

File: _output.yml [hunk 2]
```diff
@@ -16,16 +16,18 @@ bookdown::gitbook:
 bookdown::pdf_book:
   latex_engine: "pdflatex"
   citation_package: "natbib"
+  keep_tex: yes
```
Comment (benkeser) [inline _output.yml:19]: delete this. bookdown is very fragile about where `keep_tex` should go.

File: _output.yml [hunk 3]
```diff
@@ -16,16 +16,18 @@ bookdown::gitbook:
...
   highlight: tango
   toc_unnumbered: false
   toc_appendix: true
-  extra_dependencies: ["float"]
+  extra_dependencies: ["float", "flafter"]
 
 bookdown::pdf_document2:
   latex_engine: "pdflatex"
   citation_package: "natbib"
+  keep_tex: yes
```
Comment (benkeser) [inline _output.yml:30]: delete

**Review by benkeser — APPROVED** [review]

File: _output.yml (hunk 2 above)
Comment (nhejazi) [reply _output.yml]: Handled by 2990f91

File: _output.yml (hunk 3 above)
Comment (nhejazi) [reply _output.yml]: Handled by 2990f91

File: cor_coxph/code/params.R (hunk 1 above)
Comment (nhejazi) [reply cor_coxph/code/params.R]: Handled by 2990f91

**Pull request merged**